.github/workflows/helm-release.yaml

```diff
@@ -28,6 +28,11 @@ jobs:
             https://github.com/PrefectHQ/prefect.git '[!prefect-]*.*.*' | tail -n1 | sed 's/.*\///' \
           )" >> $GITHUB_OUTPUT
 
+          if [[ $PREFECT_VERSION == 3.0rc* ]]; then
```

Comment:
I dont know if this is true but do we need to do anything for regex here? Like =~ 3.0rc*?

Reply:
with double brackets `if [[ ]]`, you can do a pattern match with `*`

https://tldp.org/LDP/abs/html/comparison-ops.html

![image](https://github.com/PrefectHQ/prefect-helm/assets/19556538/6dd99c7d-9154-473e-a6cb-a657fd6028f4)

Reply:
sick TIL